tests/lax_numpy_reducers_test.py

```diff
@@ -17,6 +17,7 @@
 from functools import partial
 import itertools
 import unittest
+import pytest
```

Comment:
`pytest` is not present in all testing environments, so we need to remove this import.